src/pages/HomePage/HomePage.tsx

```diff
@@ -1,13 +1,24 @@
+import { Box } from '@material-ui/core';
 import * as React from 'react';
-
-import Button from '../../components/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
```

Comment:
To chcemy dodać tutaj? A nie do roota projektu(czyli pliku App.ts)?

Reply:
👍